.cloudbuild.yaml

```diff
@@ -198,6 +198,47 @@ steps:
   args: ['pull', 'gcr.io/ml-pipeline/workflow-controller:v3.2.3-license-compliance']
   id:   'pullArgoWorkflowController'
 
+# V2 related images
+# Prerequisite: Make sure ko image is available on the same project by running the following:
+# git clone https://github.com/GoogleCloudPlatform/cloud-builders-community.git
+# cd cloud-builders-community/ko
+# gcloud builds submit . --config=cloudbuild.yaml --project=$PROJECT_ID
+# Reference: https://dev.to/amammay/effective-go-on-gcp-lean-containers-with-ko-on-cloud-build-51ek
+
+- name: 'gcr.io/$PROJECT_ID/ko'
+  entrypoint: /bin/sh
+  args:
+    - -c
+    - |
+      cd /workspace/v2/
+      /ko publish --bare ./cmd/launcher-v2 -t $COMMIT_SHA
+  env:
+  - 'KO_DOCKER_REPO=gcr.io/$PROJECT_ID/dev/kfp-launcher-v2'
+  id:   'buildLauncherV2'
```

Comment:
I'm not sure, maybe we can consider dropping the `v2` here?

Reply:
There is another step in this cloudbuild with name `buildLauncher`, so i am adding v2. But I am going to remove the `-v2` from the container registry. Does it sound good to you?

Reply:
Because we deprecated v2 compatible, shall we remove the existing launcher step as well? Maybe as a separate PR, you can decide

Reply:
Discussed offline, I have changed the `buildLauncher` name to `buildTFJobLauncher`, and `buildLauncherV2` to `buildLauncher`.